backend/gameplay/consumers.py

```
@@ -51,19 +51,13 @@ async def increment_white_player_timer(self, chess_game_model: ChessGame, increm
 		if (increment_amount < 0):
 			return
 
-		current_time = await self.get_game_attribute(chess_game_model, "white_player_clock")
-
-		new_time = current_time + Decimal(increment_amount)
-		await self.update_game_attribute(chess_game_model, "white_player_clock", new_time, should_save=False)
+		chess_game_model.white_player_clock += Decimal(increment_amount)
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)